platform/darwin/src/MGLShapeSource.h

```diff
@@ -210,6 +210,23 @@ MGL_EXPORT
  */
 @property (nonatomic, copy, nullable) NSURL *URL;
 
+/**
+ Returns an array of map features for this source, filtered by the given predicate.
+ 
+ Each object in the returned array represents a feature for the
+ current style and provides access to attributes specified by the
+ <a href="https://www.mapbox.com/mapbox-gl-style-spec/#sources">source</a>.
```

Comment:
There’s also no need to link “source”, since this method lives inside an MGLSource subclass.